CHANGELOG.md

```diff
@@ -64,6 +64,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Added completions for [Nushell](https://nushell.sh) to `jj util completion`
 
+* Another way to resolve conflicts with Meld, in 3 tabs, adding to the
+  experimental `meld-3` tool.
```

Comment:
Let's put `meld-3` into the commit description, if it's not too much trouble.

Reply:
Sure thing.